--- a/src/nvidia/src/kernel/gpu/subdevice/subdevice_ctrl_gpu_kernel.c
+++ b/src/nvidia/src/kernel/gpu/subdevice/subdevice_ctrl_gpu_kernel.c
@@ -518,3 +518,11 @@
                 {
-                    data = NV2080_CTRL_GPU_INFO_INDEX_CMP_SKU_YES;
+                    NvU32 cmpDevId = pGpu->idInfo.PCIDeviceID >> 16;
+                    if (cmpDevId == 0x20C2 || cmpDevId == 0x2082)
+                    {
+                        data = NV2080_CTRL_GPU_INFO_INDEX_CMP_SKU_NO;
+                    }
+                    else
+                    {
+                        data = NV2080_CTRL_GPU_INFO_INDEX_CMP_SKU_YES;
+                    }
                 }
